### PR TITLE
First pass at basic examples

### PR DESCRIPTION
**Pull request by StokesMIDE**
As of the start of this PR, the example is a Jupyter Notebook in the repo 'root.' Will move and/or change format if there's a better and/or existing standard.

Since this is just the first iteration, I'm planning on merging after one approval (assuming there are no requests for changes).

### CODE REVIEW

**Comment by CFlaniganMide** [comment]
Mostly looks good, but there's an HTML tag that appears to be getting rendered as normal text on Github, which I assume is one of the targets for the readme. 
image

File: README.md [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+
+
+```python
+get_channel_table(doc)
+```
+
+
+
+
+<style type="text/css">
```
Comment (CFlaniganMide) [inline README.md:53]: These lines seem to be rendered as normal text, for some reason.

**Review by CFlaniganMide — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (StokesMIDE) [reply README.md:53]: Huh. Maybe Github disallows `<style>` tags because they could be used for malicious purposes.

Comment (CFlaniganMide) [reply README.md]:
```html
<style> 
  p {font-family: hellvetica, sans-serif;} 
</style>
```

**Review by CFlaniganMide — APPROVED** [review]

**Comment by CrepeGoat** [comment]
one more question: what's the advantage of having the tables in HTML instead of [markdown format](https://docs.inkdrop.app/manual/markdown-cheatsheet/#tables)? I thought the point of markdown & .rst files was to avoid human-illegible formatting

**Comment by StokesMIDE** [comment]
> one more question: what's the advantage of having the tables in HTML instead of [markdown format](https://docs.inkdrop.app/manual/markdown-cheatsheet/#tables)? I thought the point of markdown & .rst files was to avoid human-illegible formatting

The contents were what Python Notebook generated; I did as little editing as I could get away with, so it can easily get replaced when the ipynb gets updated. That said, I don't like the embedded HTML, either. If it ends up being problematic (like maybe on PyPI), maybe I'll download in a different format and convert with [pandoc](https://pandoc.org/) or something; it might have better output.

**Pull request merged**